.github/workflows/ci.yml

```
@@ -136,5 +136,8 @@ jobs:
       - name: Update TS version
```

Just to be fully aligned with the other addons:

```suggestion
      - name: Update TS version on addon package
```